Extensions/DialogueTree/JsExtension.js

```diff
@@ -474,18 +538,18 @@ module.exports = {
         'JsPlatform/Extensions/yarn24.png',
         'JsPlatform/Extensions/yarn32.png'
       )
-      .addParameter('string', _('Branch name'))
+      .addParameter('string', _('Branch name'), '', false)
       .getCodeExtraInformation()
       .setFunctionName('gdjs.dialogueTree.hasDialogueBranch');
 
     extension
       .addCondition(
-        'HasSelectedOptionChanged',
-        _('Has selected option changed'),
+        'SelectedOptionHasChanged',
```

Comment:
Can we keep "HasSelectedOptionChanged"? Start by the verb for a condition (Is/Has)

Reply:
The reason is that the internal name of conditions should be like functions. A function returning a "boolean" precisely. In most languages, it's a good idea to start a function name that returns a boolean by the verb:
"hasThingChanged", "isThingActivated", "shouldComponentUpdate", etc...

Reply:
changed now